hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CachePutAllBackupOperation.java

```diff
@@ -78,6 +80,16 @@ public void run() throws Exception {
     }
 
     @Override
+    public ObjectNamespace getServiceNamespace() {
```

Comment:
yes, that's a pain point. unlike map, cache operations don't have a single parent.